assets/script.js

```diff
@@ -0,0 +1,75 @@
+function main() {
+  // correctAnswer and currentAnswer used for game logic
+  const correctAnswer = ["button1", "button3", "button2"];
+  let currentAnswer = [];
+  const gameOutcomeEl = document.getElementById("gameOutcome");
+
+  // handles all the event listeners for the buttons
+  function buttonEvents() {
+    const button1El = document.getElementById("button1");
+    const button2El = document.getElementById("button2");
+    const button3El = document.getElementById("button3");
+
+    button1El.addEventListener("click", function () {
+      bringTheNoise("C4");
+      guessingGame("button1");
+    });
+    button2El.addEventListener("click", function () {
+      bringTheNoise("C5");
+      guessingGame("button2");
+    });
+    button3El.addEventListener("click", function () {
+      bringTheNoise("C6");
+      guessingGame("button3");
+    });
+  }
+  buttonEvents();
+
+  // Contains all the logic for the mini game
+  function guessingGame(button) {
+    // checks the users guess
```

Comment:
Clarification question. This part I'm unclear. What is it have an array or one lone button record.